### ISSUE TITLE
OdooGoToDeclarationHandler StringIndexOutOfBoundsException when CTRL clicking on incomplete ref in XML

### ISSUE THREAD

**Issue by jaronemo** [wt-io-it/odoo-pycharm-plugin#131]
Plugin Version: 0.6.9
IntelliJ Version: PyCharm 2022.3
IntelliJ Build: PY-223.7571.203

<details><summary>Unhandled exception in [StandaloneCoroutine{Cancelling}@3c7436fa, EDT]</summary>

```
java.lang.StringIndexOutOfBoundsException: begin 9, end -1, length 34
	at java.base/java.lang.String.checkBoundsBeginEnd(String.java:4604)
	at java.base/java.lang.String.substring(String.java:2707)
	at at.wtioit.intellij.plugins.odoo.OdooGoToDeclarationHandler.getOdooRecord(OdooGoToDeclarationHandler.java:93)
	at at.wtioit.intellij.plugins.odoo.OdooGoToDeclarationHandler.getGotoDeclarationTarget(OdooGoToDeclarationHandler.java:72)
	at com.intellij.codeInsight.navigation.actions.GotoDeclarationHandlerBase.getGotoDeclarationTargets(GotoDeclarationHandlerBase.java:14)
	at at.wtioit.intellij.plugins.odoo.OdooGoToDeclarationHandler.getGotoDeclarationTargets(OdooGoToDeclarationHandler.java:33)
	at com.intellij.codeInsight.navigation.impl.GtdProvidersKt.fromGTDProvidersInner(gtdProviders.kt:30)
	...
	at com.intellij.openapi.application.rw.InternalReadAction$runReadAction$2.invokeSuspend(InternalReadAction.kt:27)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	...
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelled}@3c7436fa, EDT]
```
</details>

**Comment by bigbear3001** [comment]
this is fixed with 0.6.11 (pending jetbrains approval) if you are eager to test it you can use the jar file from: https://github.com/wt-io-it/odoo-pycharm-plugin/tree/v0.6.11

**Comment by ap-wtioit** [comment]
0.6.11 is now available in the marketplace
